gfx/gfx_widgets.c

```diff
@@ -1477,6 +1481,11 @@ void gfx_widgets_frame(void *data)
    if (menu_screensaver_active)
       return;
 
+#ifdef HAVE_MENU
+   if (config_get_ptr()->bools.notification_show_when_menu_is_alive && !menu_state_get_ptr()->alive)
+      return;
+#endif
```

Comment:
It seems like this will completely disable the achievement popup elements, as well as things like the fastforward indicator. That may be what the user is after, but it also might be too aggressive.

Reply:
Bounty creator is indeed after this kind of "all or nothing" option, and making it more sophisticated selectability-wise will also make it a lot more complex.. Room for future improvements sure.